qiskit_ibm_runtime/api/session.py

```diff
@@ -264,6 +282,26 @@ def request(  # type: ignore[override]
         headers = self.headers.copy()
         headers.update(kwargs.pop("headers", {}))
 
+        # Set default caller
+        headers.update(
+            {"X-Qx-Client-Application": "{}/qiskit".format(CLIENT_APPLICATION)}
+        )
```

Comment:
Wouldn't this overwrite any [custom headers](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/api/session.py#L215-L222)?

Reply:
Good catch! Should be fixed in [6c0078d](https://github.com/Qiskit/qiskit-ibm-runtime/pull/733/commits/6c0078d289be112c7030d99792d17bf93820ea0f)